distributed/worker.py

```diff
@@ -785,6 +785,7 @@ async def get_metrics(self):
             executing={
                 key: now - self.tasks[key].start_time
                 for key in self.active_threads.values()
+                if key in self.tasks
```

Comment:
This line is a little worrysome. Without it, you get KeyError here during heartbeat after a failed actor action, but I don't really know why. Should the "active thread" be cleaned up somewhere?